.travis.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt whitesourceCheckPolicies whitesourceUpdate
     - stage: publish
       script: sbt publish
+    - script: cd gradle-plugin && echo "gradle.publish.key=AwHNbsVBKA20KPzWQyCRbgGQHWa9Us5Y\ngradle.publish.secret=$GRADLE_SECRET' > ~/.gradle/gradle.properties && ./gradlew publishPlugins
```

Comment:
ah nice and simple

Reply:
yeah the tricky part was testing the plugin locally, publishing itself was quite smooth!